ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/tree/node/services/TreeNodeFetcherImpl.java

```diff
@@ -20,17 +20,17 @@ public TreeNodeFetcherImpl(AppConfig appConfig, TreeNodeFactory treeNodeFactory)
 
 	@Override
 	public TreeNode getFragment(LdesFragmentRequest ldesFragmentRequest) {
-		LdesConfig ldesConfig = appConfig.getLdesConfig(ldesFragmentRequest.collectionName());
+		LdesConfig ldesConfig = appConfig.getLdesConfig(ldesFragmentRequest.viewName().getCollectionName());
```

Comment:
Minor: Whenever I see more than one dot on an object, I wonder if it's not better to create a method getCollectionName on LdesFragmentRequest